railties/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Rails 8.0.0.beta1 (September 26, 2024) ##
 
+*   Include the `immutable` directive in the `cache-control` header.
+
+    *Guillaume Briday*
+
```

Comment:
Can you move this above the `8.0.0.beta1` header (since this can't be included in the existing release)?

Reply:
Oops yeah obviously, I pushed the fix